.github/actions/deploy_v2/action.yml

```diff
@@ -0,0 +1,129 @@
+name: deploy_v2
+description: deploys application
+
+inputs:
+  arm-access-key:
+    required: true
+  azure-credentials:
+    required: true
+  environment:
+    description: Environment to deploy to
+    required: true
+  pr-number:
+    description: PR number for the review app
+    required: false
+  sha:
+    description: Commit sha to be deployed
+    required: true
+  slack-webhook:
+    required: true
+
+outputs:
+  deploy-url:
+    value: ${{ steps.set_env_var.outputs.deploy_url }}
+
+runs:
+  using: composite
+  steps:
+    - name: Set Environment variables
+      id: set_env_var
+      shell: bash
+      run: |
+        tf_vars_file=terraform/workspace_variables/${{ inputs.environment }}.tfvars.json
+        cluster=$(jq -r '.cluster' ${tf_vars_file})
+
+        if [ -n "${{ inputs.pr-number }}" ]; then
+          echo "DEPLOY_ENV=review_aks-${{ inputs.pr-number }}" >> $GITHUB_ENV
+          echo "deploy_url=https://apply-review-${{ inputs.pr-number }}.${cluster}.teacherservices.cloud" >> $GITHUB_OUTPUT
+        else
+          echo "DEPLOY_ENV=${{ inputs.environment }}" >> $GITHUB_ENV
+
+          hostname=$(jq -r '.service_gov_uk_host_names[0]' ${tf_vars_file})
+          if [[ $hostname != null ]]; then
+            echo "deploy_url=https://${hostname}.apply-for-teacher-training.service.gov.uk" >> $GITHUB_OUTPUT
+          else
+            echo "deploy_url=https://apply-${{ inputs.environment }}.${cluster}.teacherservices.cloud" >> $GITHUB_OUTPUT
```

Comment:
This will not work for the production cluster as the URL is `https://apply-${{ inputs.environment }}.teacherservices.cloud`. Can we amend the logic here?

Reply:
Added some logic when the cluster is production we don't add the cluster name to the deploy_url